fighthealthinsurance/rest_serializers.py

```
@@ -525,3 +534,128 @@ class SelectContextArticlesSerializer(serializers.Serializer):
+
+    @extend_schema_field(DictionaryStringField)
+    def get_questions(self, obj):
+        """Format questions for display."""
+        if not obj.questions:
+            return {}
+        try:
+            questions = {}
+            for k, v in obj.questions:
+                questions[k] = v
+            return questions
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)